api/pkg/db/model/model.go

```diff
@@ -73,8 +73,16 @@ type (
 
 	User struct {
 		gorm.Model
+		Name        string
```

Comment:
Here don't we need `not null` constraints ??

Reply:
`Name` could be `NULL`, for a normal user `GithubName` will set and `Name` will be NULL